src/openzaak/conf/includes/base.py

```diff
@@ -280,6 +280,7 @@
             "level": "DEBUG",
             "propagate": False,
         },
+        "vng_api_common": {"handlers": ["console"], "level": "INFO", "propagate": True},
```

Comment:
@sergei-maertens it seemed that logging from vng_api_common did not work yet, so I added it to openzaak